mtp_cashbook/apps/mtp_auth/forms.py

```diff
@@ -7,31 +8,63 @@
 from mtp_common.user_admin.forms import SignUpForm
 from oauthlib.oauth2 import OAuth2Error
 from requests import RequestException
+from zendesk_tickets.forms import BaseTicketForm
 
 logger = logging.getLogger('mtp')
 
 ACCOUNT_REQUEST_ROLE = 'prison-clerk'
 
 
-class CashbookSignUpForm(SignUpForm):
+class CashbookSignUpForm(SignUpForm, BaseTicketForm):
     prison = forms.ChoiceField(label=_('Prison'), help_text=_('Enter the prison you’re based in'))
 
+    # Non form class variables
+    cashbook_account_request_zendesk_subject = 'Requesting a new staff account - Digital Cashbook'
```

Comment:
for the other tickets, we use the subject line to also say which team is supposed to answer it:
https://github.com/ministryofjustice/money-to-prisoners-cashbook/blob/b62fcd3922481b651c3d5e303a8b20efc3dc6057/mtp_cashbook/apps/feedback/views.py#L12
vs
https://github.com/ministryofjustice/money-to-prisoners-send-money/blob/44138128836114592a99bc347fc3573c142d2773/mtp_send_money/apps/help_area/views.py#L27
so you think it's worth saying something similar?